contrib/node/src/python/pants/contrib/node/tasks/node_resolve.py

```diff
@@ -143,7 +143,7 @@ def execute(self):
             if not vt.valid:
               resolver_for_target_type = self._resolver_for_target(target).global_instance()
               resolver_for_target_type.resolve_target(self, target, vt.results_dir, node_paths)
-            node_paths.resolved(target, vt.results_dir)
+              node_paths.resolved(target, vt.results_dir)
```

Comment:
What is the impact of this statement? Was it a bug previously?

Reply:
Doesn't change the outcome of tests so I changed it back